src/sqlfluff/rules/L046.py

```diff
@@ -45,25 +45,19 @@ def _get_whitespace_ends(s):
         pos = main.find(inner)
         return main[:pos], inner, main[pos + len(inner) :]
 
-    def _eval(self, segment, templated_file, memory, **kwargs):
-        # Extract some data from the segment. Importantly, all
-        # of these have defaults in case we don't have an
-        # enriched position marker.
-        source_slice = getattr(segment.pos_marker, "source_slice", None)
-        is_literal = getattr(segment.pos_marker, "is_literal", None)
```

Comment:
Was this variable (`is_literal `) actually used anywhere? I see its removal but no corresponding uses of it being removed.

Reply:
Yeah it was used in line 56 below, but I'm now calling `.is_literal()` directly.